app/services/email_marketer_service/colossus/fetch_lists.rb

```diff
@@ -0,0 +1,21 @@
+module EmailMarketerService
+  module Colossus
+    class FetchLists
+      attr_reader :account
+
+      def initialize(account:)
+        @account = account
+      end
+
+      def call
+          account.lists.find_or_create_by(
+            list_id: 1676,
```

Comment:
Again, I don't see how this makes sense. This is a whole service for creating single hard-coded record, maybe we should use database seeds instead.